src/main/java/org/jenkins/plugins/lockableresources/actions/LockableResourcesRootAction.java

```diff
@@ -112,7 +121,9 @@ public LockableResource getResource(final String resourceName) {
    * @return Amount of free labels.
    */
   public int getFreeResourceAmount(String label) {
```

Comment:
check where it is used and set it to deprecated
This operation will be very slowly (because of synchronized over all resources)